src/main/java/com/unosquare/adminCore/ApplicationStartup.java

```diff
@@ -0,0 +1,45 @@
+package com.unosquare.adminCore;
+
+import com.unosquare.adminCore.entity.EmployeeUserRole;
+import com.unosquare.adminCore.enums.EmployeeUserRoles;
+import com.unosquare.adminCore.payload.SignUpRequest;
+import com.unosquare.adminCore.repository.EmployeeUserRoleRepository;
+import com.unosquare.adminCore.service.EmployeeService;
+import com.unosquare.adminCore.service.EmployeeUserRoleService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.event.ApplicationReadyEvent;
+import org.springframework.context.ApplicationListener;
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDate;
+
+@Component
+public class ApplicationStartup
+        implements ApplicationListener<ApplicationReadyEvent> {
+
+    @Autowired
+    EmployeeService employeeService;
+
+    @Autowired
+    EmployeeUserRoleService employeeUserRoleService;
```

Comment:
Yeah lets change this to EmployeeRole, it sounds like it's joining employee, user and roles